backend/.env.example

```diff
@@ -1,2 +1,3 @@
 GIPHY_API_KEY=<YOUR_API_KEY>
-PIXABAY_API_KEY=<YOUR_API_KEY>
\ No newline at end of file
+PIXABAY_API_KEY=<YOUR_API_KEY>
+UNSPLASH_API_KEY=<YOUR_API_KEY>
```

Comment:
Please add new line at the end of each pages. There should be a configuration on your IDE to do it for you.